src/tests/ftest/daos_test/daos_core_test.py

```diff
@@ -270,6 +270,21 @@ def test_daos_drain_simple(self):
         """
         self.run_subtest()
 
+    def test_daos_extend_simple(self):
+        """Jira ID: DAOS-1568
+
+        Test Description:
+            Run daos_test -B
+
+        Use cases:
+            Core tests for daos_test
+
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,ib2,medium
+        :avocado: tags=daos_test,daos_core_test,test_daos_drain_simple
```

Comment:
Use test_daos_extend_simple insytead of test_daos_drain_simple

Reply:
ah, right. Thanks.